hack/lib/mesh.bash

```diff
@@ -0,0 +1,201 @@
+kind: Subscription
+metadata:
+  name: kiali-ossm
+  namespace: openshift-operators
+spec:
+  channel: stable
+  name: kiali-ossm
+  source: ci-operators
+  sourceNamespace: openshift-operators
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: servicemeshoperator
+  namespace: openshift-operators
+spec:
+  channel: "1.0"
+  name: servicemeshoperator
+  source: ci-operators
+  sourceNamespace: openshift-operators
+EOF
+
+  logger.info "Waiting until service mesh operators are available"
+  timeout 600 "[[ \$(oc get deploy -n openshift-operators ${mesh_deployments} --no-headers | wc -l) != 4 ]]" || return 1
+  oc wait --for=condition=Available deployment ${mesh_deployments} --timeout=300s -n openshift-operators || return $?
+}
+
+
+function deploy_smcp {
+  oc create namespace istio-system -o yaml --dry-run | oc apply -f -
+
+  cat <<EOF | oc apply -f - || return $?
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic-install
+  namespace: istio-system
+spec:
+
+  istio:
+    global:
+      proxy:
+        resources:
+          requests:
+            cpu: 100m
+            memory: 128Mi
+          limits:
+            cpu: 500m
+            memory: 128Mi
+
+    gateways:
+      istio-egressgateway:
+        enabled: false
+      istio-ingressgateway:
+        enabled: false
+
+    mixer:
+      policy:
+        autoscaleEnabled: false
+      telemetry:
+        autoscaleEnabled: false
+        resources:
+          requests:
+            cpu: 100m
+            memory: 1G
+          limits:
+            cpu: 500m
+            memory: 4G
+
+    pilot:
+      autoscaleEnabled: false
+    kiali:
+      enabled: true
+    grafana:
+      enabled: true
+    tracing:
+      enabled: true
+    prometheus:
+      enabled: true
+EOF
+
+  logger.info "Waiting until service mesh deployments are available"
+  timeout 600 "[[ \$(oc get deploy -n istio-system ${istio_deployments} --no-headers | wc -l) != 5 ]]" || return 1
+  oc wait --for=condition=Available deployment ${istio_deployments} --timeout=300s -n istio-system || return $?
+}
+
+
+# add_smmr adds ServiceMeshMemberRoll, Networkpolicy and label.
+# It needs to call after deploy knative-serving.
+function add_smmr {
+  cat <<EOF | oc apply -f -
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+  namespace: istio-system
+spec:
+  members:
+    # a list of projects joined into the service mesh
+    - default
+EOF
+
+  cat <<EOF | oc apply -f -
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-from-serving-system-namespace
+  namespace: default
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          serving.knative.openshift.io/system-namespace: "true"
+  podSelector: {}
+  policyTypes:
+  - Ingress
+EOF
+
+  oc label namespace knative-serving serving.knative.openshift.io/system-namespace=true --overwrite         || true
```

Comment:
Didn't we add those to the namespaces already anyway? :thinking:

Reply:
No, `knative-serving` namespace is created by users not operator so we decided that we don't manipulate it by our operator.
And the same label is added by our test script (`test/lib.bash`) but it is not called during `make install`